pages/platform/databases/databases_09_order_terraform/guide.de-de.md

```diff
@@ -243,9 +242,17 @@ export URI=$(terraform output -raw cluster_uri)
 
 And, voilà, the MongoDB cluster is created.
 
-## Go further
+## How to with other databases?
```

Comment:
Isn't a verb missing here ?

Reply:
In that particular turn of phrase, the verb is redundant